### DISCUSSION TITLE
2016/06/26/beacons-62-and-64-in-eastern-pa

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#11] [Announcements]
# 2016/06/26/beacons-62-and-64-in-eastern-pa

http://localhost:4000/2016/06/26/beacons-62-and-64-in-eastern-pa

**Comment by jengalas** [comment]
Posted by Trevor Jones
October 18, 2022 at 8:39 pm
_Imported from WordPress_

I know of beacon 64 well .. I do the restoration work of the mountain..I have some bulbs from the lights. I actually got a call from Washington to guide up some faa agents to find it for them …wish I had a picture

**Comment by jengalas** [comment]
Posted by rwaeger
September 18, 2016 at 5:18 pm
_Imported from WordPress_

This is for Mr Pastore….I’m helping an Eagle scout to identify a “beacon” that used to be located near CXY on Ressers summit New Cumberland.
Do you have any knowledge or info.

rwaeger AOPA member

**Reply by jengalas** [reply]
Posted by Zhanna
October 10, 2016 at 5:39 pm
_Imported from WordPress_

You asked about a beacon in New Cumberland, PA, near airport CXY. I think it may be this one:

[http://www.ngs.noaa.gov/cgi-bin/ds_mark.prl?PidBox=KW3105](http://www.ngs.noaa.gov/cgi-bin/ds_mark.prl?PidBox=KW3105)

You can easily view its location on my map of airway beacons here (just search for the PID KW3105):

[https://thesurveystation.com/map-of-ngs-airway-beacons/](https://thesurveystation.com/map-of-ngs-airway-beacons/)

As you can see from the 2006 report, it seems that the beacon tower is gone but the concrete foundation remains. It was apparently still in use in 1934 but the beacon light was removed by 1950. Unfortunately, that’s all the information I have. I forwarded your question to Mr. Pastore, but he doesn’t have any further information.
